### PR TITLE
shared data and username emphasis

### LINKED ISSUE #101: Document sharing a folder between users
Possibly with `/etc/skel`?

### LINKED ISSUE #102: Install on your own server install script doesn't work when copied
I believe that the installation instructions won't work on the "custom server" page because if you copy/paste it into a terminal then the first line is treated as a comment and nothing is run.

I think instead, we should tell people something like:

1. Create a file called `install.sh`
   
   `nano install.sh`

2. Paste in the following lines

   ```
   #!/bin/bash
   curl https://raw.githubusercontent.com/jupyterhub/the-littlest-jupyterhub/master/bootstrap/bootstrap.py | sudo python3 - --admin <admin-user-name>
   ```

3. Modify `<admin-user-name>` to be your user name
4. Exit and save: `ctrl+X`, "yes", `enter`
5. Run the install script:
    
    `sh install.sh`

what do you think @yuvipanda ?

### PR DESCRIPTION
**Pull request by choldgraf**
This is some wording to show people how to distribute data with their users. Let me know what you think!

closes #101 
closes #102 (by a rebase accident? idk!)

### CODE REVIEW

File: docs/howto/share-data.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+If your data is large or you don't want copies of it to exist, you can create
+a read-only shared folder that users have access to. To do this, follow these
+steps:
+
+#. Log in to your JupyterHub as an **administrator user**.
+#. Create a terminal session within your JupyterHub interface.
+#. Create a folder where your data will live:
+
+   ```
```
Comment (yuvipanda) [inline docs/howto/share-data.rst]: This is markdown, no?

File: docs/howto/share-data.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+   mkdir /srv/data/mydatafolder
+   ```
+#. Download the data into this folder. For example, using `curl` or by running
+   a `python` script that downloads the data.
```
Comment (yuvipanda) [inline docs/howto/share-data.rst]: Also markdown.

File: docs/howto/share-data.rst [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+follow these steps:
+
+#. Change into the skeleton directory:
+
+   ```
+   cd /etc/skel
+   ```
+#. Create a symbolic link to the data folder
+
+   ```
```
Comment (yuvipanda) [inline docs/howto/share-data.rst]: more markdown :)

File: docs/howto/share-data.rst [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+   ln -s /src/data/mydatafolder mydatafolder
```
Comment (yuvipanda) [inline docs/howto/share-data.rst]: needs sudo

File: docs/howto/share-data.rst [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+If your data is large or you don't want copies of it to exist, you can create
+a read-only shared folder that users have access to. To do this, follow these
+steps:
+
+#. Log in to your JupyterHub as an **administrator user**.
+#. Create a terminal session within your JupyterHub interface.
+#. Create a folder where your data will live:
+
+   ```
+   mkdir /srv/data/mydatafolder
```
Comment (yuvipanda) [inline docs/howto/share-data.rst]: needs sudo

File: docs/howto/share-data.rst [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+   ```
+#. Download the data into this folder. For example, using `curl` or by running
```
Comment (yuvipanda) [inline docs/howto/share-data.rst]: They need to use sudo for this too.

**Comment by yuvipanda** [comment]
@choldgraf when it is changes like this, do I have a general ok from you to just make the fixes and merge? I know we have it for other projects, but wanted to explicitly check :)

**Comment by yuvipanda** [comment]
@choldgraf I've implemented all the changes and am gonna merge soon. LMK if you don't want me to do this in the future :)

**Comment by choldgraf** [comment]
@yuvipanda sorry I missed your last comment, yes you definitely have permission from me to do this!

**Comment by choldgraf** [comment]
(if I disagree, I'll just tell you I disagree and/or open a PR :-) )

**Pull request merged**